### ISSUE TITLE
entity_picture contains invalid URL if source is YT

### ISSUE THREAD

**Issue by Switch123456789** [bushvin/hass-integrations#31]
If I play music from Youtube, home assistant can't show the cover art because the entity_picture-value leads to a non existing file. Apologies if this is an issue with mopidy and not the integration.

**Comment by bushvin** [comment]
Hello @Switch123456789 

Mopidy is an audio media player, not a video media player, so it would make sense not showing any video playback.

Will investigate a bit more

**Comment by Switch123456789** [comment]
Hi @bushvin,
thank you for investigating! I only refer to the cover-art/thumbnail (not video-playback of any kind). On Iris, I can see it and the file is accessible but the `entity_picture`-attribute of the `media_player`-entity refers to some very different URL that leads to nowhere. If I play local files everything works as it should.

Let me know if you want more information or if I can assist in any other way. =)

**Comment by bushvin** [comment]
@Switch123456789 

Can you please tell me which Mopidy Extension you use for youtube? and the version installed?
The youtube extension entry in your mopidy.conf would be handy as well.

**Comment by Switch123456789** [comment]
Sure!
mopidy `3.4.1-1`
Mopidy-YouTube `3.7`
yt-dlp `2023.7.6`

Output of `mopidyctl config`:
```
[youtube]
enabled = true
allow_cache = true
youtube_api_key = 
search_results = 32
playlist_max_videos = 20
api_enabled = false
channel_id = 
musicapi_enabled = false
musicapi_cookie = 
musicapi_cookiefile = 
autoplay_enabled = false
strict_autoplay = false
max_autoplay_length = 600
max_degrees_of_separation = 3
youtube_dl_package = yt_dlp
```
`mopidy.conf`:
```
[youtube]
allow_cache = true
search_results = 32
youtube_dl_package = yt_dlp
musicapi_enabled = false
```

**Comment by bushvin** [comment]
Hey @Switch123456789 

I found the origin of the issue. Should be solved with #39 

Thanks for your patience!

**Comment by Switch123456789** [comment]
I just checked and it works fine so far! Thank you sir :)
